components/notification/demo/duration.md

```diff
@@ -1,23 +1,32 @@
 ---
 order: 1
-title: 自动关闭的延时
+title: 
+  zh-CN: 自动关闭的延时
+  en-US: A duration to close notification box 
 ---
 
+## zh-CN
+
 自定义通知框自动关闭的延时，默认`4.5s`，取消自动关闭只要将该值设为 `0` 即可。
 
+## en-US
+
+To customize a duration to close notification, default value is `4.5s`, if you set the value to `0`,
```

Comment:
`duration` can be used to specify how long the notification is open. After the duration time elapses, the notification closes automatically. If not specified, default value is `4.5s`. If you set the value to `0`,